### PR TITLE
Log at INFO level by default (including in autoscaler).

### PR DESCRIPTION
**Pull request by ericl**
## What do these changes do?

Before this change, the autoscaler `up` and related commands don't print any info messages to the console at all. This was a regression from 0.5.
Is this the right way to set the defaults? @richardliaw @robertnishihara 

## Related issue number

https://github.com/ray-project/ray/issues/2812

### CODE REVIEW

File: python/ray/scripts/scripts.py [hunk 1]
```diff
@@ -567,6 +567,9 @@ def get_head_ip(cluster_config_file, cluster_name):
 
 
 def main():
+    logging.basicConfig(
```
Comment (ericl) [inline python/ray/scripts/scripts.py]: @mitar I trust this the right way to initialize logging?

**Comment by mitar** [comment]
I think this should probably be a command line argument to configure both of these? [See here for example](https://github.com/ray-project/ray/blob/master/python/ray/scripts/scripts.py#L164).

File: python/ray/scripts/scripts.py (hunk 1 above)
Comment (mitar) [reply python/ray/scripts/scripts.py]: Yes.

**Comment by ericl** [comment]
I moved it to a common option for the cli group. This does mean that now you do `ray --logging-level=X start` instead of `ray start --logging-level=X` due to click.

**Comment by mitar** [comment]
Even better!

**Review by richardliaw — APPROVED** [review]

**Pull request merged**